test/campaigns_controllers.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
The original intent was for it to test all `campaigns` controllers, but you make a fair point. We should probably have a separate test module for each controller for better maintainability. I've addressed this in 81b5c8d